packages/react-native-legal/package.json

```diff
@@ -35,7 +35,6 @@
     "ios",
     "lib",
     "plugin",
-    "plugin-utils",
```

Comment:
currently this will break the library, as there're certain platform utilities that are used both in bare and expo plugin and are not migrated to the shared package

- `react-native-legal/plugin-utils/src/android.ts`
- `react-native-legal/plugin-utils/src/ios.ts`

Reply:
Right, that's a mistake on my side - I originally thought about migrating these to the shared package, but they're only used in the plugins (native-specific). It's a good catch - this is a change we'd have to revert to match the current approach.

Just looking at the other approach which I discarded, @mateusz1913  would you see any benefits in moving these utils to the shared package?

Reply:
Those platforms utils seem to have no real usage outside of the RNLegal scope, so we can keep them inside of RNLegal code. But moving them to shared package seems to be fine as well, considering that the RNLegal plugins already import some utilities from it, so  it is up to you

Reply:
Resolved in cf431d35635458e69830dbb936f8aa2abbf51b77